flink/src/main/scala/org/apache/mahout/flinkbindings/FlinkEngine.scala

```diff
@@ -249,12 +249,10 @@ object FlinkEngine extends DistributedEngine {
   override def drmBroadcast(v: Vector)(implicit dc: DistributedContext): BCast[Vector] =
     FlinkByteBCast.wrap(v)
 
-
   /** Broadcast support */
   override def drmBroadcast(m: Matrix)(implicit dc: DistributedContext): BCast[Matrix] =
     FlinkByteBCast.wrap(m)
 
-
   /** Parallelize in-core matrix as flink distributed matrix, using row ordinal indices as data set keys. */
   override def drmParallelizeWithRowIndices(m: Matrix, numPartitions: Int = 1)
```

Comment:
we should probably note in the comments that this breaks the drmParallelize() contract, and ignores the numPartitions parameter.